js/actions/appActions.js

```diff
@@ -1834,10 +1834,11 @@ const appActions = {
    * Dispatches a message to add a given publisher to the ledger.
    * @param location - the URL of the publisher
    */
-  addPublisherToLedger: function (location) {
+  addPublisherToLedger: function (location, tabId = false) {
```

Comment:
This shouldn't be false, where we are expecting number for this param. I think default should be `tabState.TAB_ID_NONE`